integration_tests/test_suites/k8s-test-suite/tests/test_executor.py

```diff
@@ -522,6 +523,8 @@ def test_k8s_executor_owner_references_garbage_collection(
         time.sleep(1)
 
 
+# K8s issue for Python 3.12 and 3.13 not supported https://github.com/kubernetes-client/python/issues/2419
+@pytest.mark.skipif(sys.version_info >= (3, 12), reason="Python 3.12 and 3.13 not supported")
```

Comment:
i don't think that issue is actually related here - i think the test is just flaky and in general the k8s api does work fine on python 3.12

Reply:
Should we drop this then?